### PR TITLE
[14.0][FIX] stock_barcodes avoid method action_create_move reading product …

### PR DESCRIPTION
**Pull request by Bilbonet**
The situation explained here #463

### CODE REVIEW

**Comment by Bilbonet** [comment]
Can anybody review this?
Thanks

**Comment by rousseldenis** [comment]
@sergio-teruel @LoisRForgeFlow @pedrobaeza @chienandalu

File: stock_barcodes/static/src/js/basic_controller.js [hunk 1]
```diff
@@ -283,13 +283,8 @@ odoo.define("stock_barcodes.BasicController", function (require) {
                             const $action_confirm = self.$(
                                 "button[name='action_confirm']:visible"
                             );
-                            const $action_confirm_force = self.$(
```
Comment (pedrobaeza) [inline stock_barcodes/static/src/js/basic_controller.js:286]: Why are you removing these features?

Comment (Bilbonet) [reply stock_barcodes/static/src/js/basic_controller.js:286]: In the context of doing a picking. 
The function that was called from these actions, forced the creation a product movement when you scanned a wrong product.
As I explain in the PR, I think is better not doing nothing if the product scaned is not in the picking list.

Comment (pedrobaeza) [reply stock_barcodes/static/src/js/basic_controller.js:286]: I don't think so. We may need to add extra products in the picking. That's an intended feature.

**Comment by pedrobaeza** [comment]
Closing as stated in https://github.com/OCA/stock-logistics-barcode/issues/463#issuecomment-1518604393

File: stock_barcodes/static/src/js/basic_controller.js (hunk 1 above)
Comment (Bilbonet) [reply stock_barcodes/static/src/js/basic_controller.js:286]: OK, you're right 
Now it works fine. 
Thanks for the comments

**Pull request closed without merging**